api/test/unit/routes/policies.test.js

```diff
@@ -31,7 +31,7 @@ describe('Routing Policy Creation', function() {
 
   it('should create a policy for app id 12345', function(done) {
     nock(schedulerURI)
-    .put('/v2/schedules/12345')
+    .put(/\/v2\/schedules\/12345\?guid=.*/)
```

Comment:
I just want to understand why you didn't use query function like following code?

```
  nock(schedulerURI)
    .put('/v2/schedules/12345')
    .query({guid: '.*'})
```